CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 # Changelog
 
-## Unreleased
+## [1.0.4](https://github.com/lostisland/faraday-multipart/releases/tag/v1.0.3) (2022-06-07)
```

Comment:
no big deal, but the link points to v1.0.3

Reply:
Fixing now